docs/References.bib

```diff
@@ -97,9 +97,9 @@ @article{Fishbone1976apj
 
 @article{Foucart2015vpa,
       author         = "Foucart, Francois and O'Connor, Evan and Roberts, Luke
-                        and Duez, Matthew D. and Haas, Roland and Kidder, Lawrence
-                        E. and Ott, Christian D. and Pfeiffer, Harald P. and
-                        Scheel, Mark A. and Szilagyi, Bela",
```

Comment:
For the record, in #1353 I suggested making the indentation here consistent with the rest of the file (2 spaces) while you are touching this

Reply:
I've added a relevant issue: #1381